cmd/create/cluster/cmd.go

```diff
@@ -169,6 +172,27 @@ func init() {
 			"multizone clusters need at least 3 nodes.",
 	)
 
+	flags.BoolVar(
+		&args.autoscalingEnabled,
+		"enable-autoscaling",
+		false,
+		"Enable autoscaling of compute nodes.",
+	)
+
+	flags.IntVar(
+		&args.minReplicas,
+		"min-replicas",
+		0,
```

Comment:
Maybe we should default to `2` here, otherwise since there's no other indication the user might enter 0 or 1. We can treat this in the same way as the compute nodes, where the hard-coded default is 2, but we update it to 3 in the code if they select multi-az. This reduces the possibility of input errors.

Reply:
Done